cmd/insprctl/cli/broker.go

```diff
@@ -0,0 +1,92 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"inspr.dev/inspr/pkg/cmd"
+	"inspr.dev/inspr/pkg/cmd/utils"
+	"inspr.dev/inspr/pkg/ierrors"
+)
+
+// NewBrokerCmd creates broker command for Inspr CLI
+func NewBrokerCmd() *cobra.Command {
+
+	kafkaCmd := cmd.NewCmd("kafka").
+		WithDescription("Configures a kafka broker on insprd by importing a valid yaml file carring configurations for one of the supported brokers").
+		WithExample("kafka kafka.yaml", "brokers kafka <file>").
+		ExactArgs(1, kafkaConfig)
+
+	return cmd.NewCmd("brokers").
+		WithDescription("Retrieves brokers currently installed").
+		WithLongDescription(`Broker takes a command of (brokers) to get installed brokers,
+and a subcommand of (kafka) to install a new kafka broker`).
+		WithExample("get brokers", "brokers").
+		AddSubCommand(kafkaCmd).
+		NoArgs(getBrokers)
+}
+
+func getBrokers(ctx context.Context) error {
+	client := utils.GetCliClient()
+	out := utils.GetCliOutput()
+	resp, err := client.Brokers().Get(context.Background())
+	if err != nil {
+		utils.RequestErrorMessage(err, out)
+		return err
+	}
+
+	fmt.Fprintf(out, "DEFAULT:\n%s\n", resp.Default)
+	fmt.Fprintln(out, "AVAILABLE:")
+	lines := make([]string, 0)
+	for _, broker := range resp.Available {
+		printLine(broker, &lines)
+	}
+	printTab(&lines)
+	return nil
+}
+
+func kafkaConfig(c context.Context, args []string) error {
+	return brokerConfig("kafka", args[0])
+}
+
```

Comment:
there could an error on this function in which the args slice is empty, therefore is a valid to check if he has at least one element, otherwise return an error